### ISSUE TITLE
Select multiple pass holders?

### ISSUE THREAD

**Issue by dhimmel** [coltonc3/shred-bot#3]
Thanks for sharing shred-bot. 🎿 

Would it be possible to have an option to select multiple passholders? Currently, it appears to create a reservation for the primary passholder only, akin to the GUI selection of:

![image](https://user-images.githubusercontent.com/1117703/106842004-30bd9300-6671-11eb-8bae-3b2df9f71788.png)

**Comment by coltonc3** [comment]
@dhimmel Thanks for the help! I wasn't sure if this would catch anyone's eye. 

What page is this on? Date selector page? I'm thinking I'll need another field in `creds.rb` for which pass holders to select.

**Comment by dhimmel** [comment]
> I wasn't sure if this would catch anyone's eye.

Indeed! I made a similar but stupider bot once for hiking permits called [hackjohn](https://github.com/dhimmel/hackjohn) (now deceased). In time they will come! I also found another project [dude_wheres_my_lift_ticket](https://github.com/jeffgebhardt/dude_wheres_my_lift_ticket), but it doesn't seem like you can select your mountain yet there.

> What page is this on?

https://account.ikonpass.com/en/myaccount/add-reservations/ > 2. CHOOSE YOUR DATES > SELECT PASS HOLDERS

Here's a bit more of the screen:

![image](https://user-images.githubusercontent.com/1117703/106901965-4b6e2700-66c6-11eb-9fd2-06f9860a9304.png)

Is the interface different for accounts without multiple passholders?

Let me know if I can provide anything additional to help here.
